stablehlo/transforms/Passes.td

```diff
@@ -28,7 +28,7 @@ def VhloLegalizeToStablehloPass : Pass<"vhlo-legalize-to-stablehlo", "ModuleOp">
 def VhloToVersionPass : Pass<"vhlo-to-version"> {
   let summary = "Convert between versions of VHLO.";
   let options = [
-    Option<"targetVersion", "target", "std::string", "",
+    Option<"targetVersionClopt", "target", "std::string", "",
```

Comment:
"targetVersionOption"? I don't think using an abbreviation here is worth it, since there's a straightforward conventional name which is just one character longer.